src/webhint-theme/source/core/css/navigation.css

```diff
@@ -165,10 +165,6 @@ button.mobile-search-button {
     background-size: 2rem;
 }
 
-[aria-hidden=true] {
-    display: none;
-}
```

Comment:
What's the browser support matrix for the website? I ask because `hidden` support was introduced in IE11. If we are trying to support anything prior to that we may want to change this to:
```css
[hidden] {
    display: none;
}
```

Reply:
We can probably add this piece of code then.